app/javascript/components/lease_ups/LeaseUpApplicationsFilter.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Form, Text, Select } from 'react-form'
+import formUtils from '~/utils/formUtils'
+import Loading from '~/components/molecules/Loading'
+
+const StatusOptions = formUtils.toOptions(['No Status', 'Processing', 'Disqualified', 'Approved', 'Lease Signed', 'Waitlisted', 'Withdrawn', 'Appealed', [null, 'All Status']])
```

Comment:
These are defined in app/javascript/utils/statusUtils.js (aside from 'No Status' and 'All Status')

Reply:
I was looking for that! Thanks :)